LuckySpin/Controllers/SpinnerController.cs

```diff
@@ -104,16 +80,12 @@ public IActionResult Index(Player player)
         /***
          * ListSpins Action
          **/
-
          public IActionResult LuckList(int id)
         {
-            //TODO: use the id to get the current player, including her Spins list
-
+            var Player = _dbc.Players.Include(x => x.Spins).Single(x => x.Id == id);
```

Comment:
Generally we will use lowercase for the instance, and uppercase for the class.
var player = _dbc.Players.Included...